ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+from random import randint
+
+class Player:
+    def __init__(self, player_name, player_token):
+        self.name = player_name
+        self.token = player_token
+        self.money = 1500
+        self.properties = []
+        self.position = 0
+        self.in_jail = False
+        self.is_turn = False
+    
+    def __repr__(self):
```

Comment:
We generally use `__repr__` to print a representation of the object for logging/debugging purposes rather than printing the actual user-visible text. It's not _bad_ to do this, though, just uncommon.

Reply:
Mm, I see, the course didn't really make that apparent. I just thought it was a convenient way to see all the info of a class.